scripts/run-ft-benchmark.py

```diff
@@ -52,9 +52,19 @@ def main() -> None:
 
     args = parser.parse_args()
 
+    time_seconds = args.time
```

Comment:
I wonder if we can use some library for this? E.g. there is `parse_timespan` in Locust: https://github.com/locustio/locust/blob/1fc3143f1f275c6b311c9cecfab151410a76467e/locust/util/timespan.py#L5